MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include README.md LICENSE.md CHANGELOG.md
 include src/sqlfluff/config.ini
+include src/sqlfluff/py.typed
```

Comment:
Including this marker file allows for mypy to pick up the simple api type hints when sqlfluff is installed as a dependency to another project.

Reply:
This is now moved to setup.cfg